editor/gui/touch_actions_panel.cpp

```diff
@@ -53,10 +56,27 @@ void TouchActionsPanel::_notification(int p_what) {
 			delete_button->set_button_icon(get_editor_theme_icon(SNAME("Remove")));
 			undo_button->set_button_icon(get_editor_theme_icon(SNAME("UndoRedo")));
 			redo_button->set_button_icon(get_editor_theme_icon(SNAME("Redo")));
+			cut_button->set_button_icon(get_editor_theme_icon(SNAME("ActionCut")));
+			copy_button->set_button_icon(get_editor_theme_icon(SNAME("ActionCopy")));
+			paste_button->set_button_icon(get_editor_theme_icon(SNAME("ActionPaste")));
 		} break;
 	}
 }
 
+void TouchActionsPanel::input(const Ref<InputEvent> &event) {
+	if (ctrl_btn_pressed && event->has_method("set_ctrl_pressed")) {
+		event->call("set_ctrl_pressed", true);
```

Comment:
```suggestion
		event->call(SNAME("set_ctrl_pressed"), true);
```
I think this is going to be called quite often.
Same in others.